### DISCUSSION TITLE
Dynamic form recipients

### DISCUSSION THREAD

**Discussion by tidygraphic** [statamic/cms#5929] [Troubleshooting]
We've created a number of forms for a new site we're building and they live on 'people' pages throughout the site. Each person has a different email address but the form (fields etc) are the same. So, we've create one 'quick_contact' form which is added to each persons page. This seems the cleanest way to do things. 

However, we'd like each form to be emailed to the main contact on that persons page. So, quick contact form for person X gets emailed to person X when submitted. 

I've done some Googling and it appears a couple of times, with [this being the suggested solution](https://github.com/statamic/v2-hub/issues/450). However I'm hoping to avoid exposing the email address (even in a hidden form field) as it is easily manipulated on the front end and so leaves a security hole in the site. 

Is there a better solution to this problem other than to create lots of duplicate forms per recipient (we could potentially have hundreds of people listed, so this would be a monstrous list of forms).

**Comment by edalzell** [comment]
You could use their user id in the form (so that would be public), then make a listener that sent the email?  You'd turn off the built in emailing, of course.
